ant-networking/src/replication_fetcher.rs

```diff
@@ -50,6 +50,13 @@ pub(crate) struct ReplicationFetcher {
     /// used when the node is full, but we still have "close" data coming in
     /// that is _not_ closer than our farthest max record
     farthest_acceptable_distance: Option<Distance>,
+    /// Scoring of peers collected from storage_challenge.
+    /// To be a trustworthy replication source, the peer must has two latest scoring both healthy.
+    peers_scores: HashMap<PeerId, (VecDeque<bool>, Instant)>,
```

Comment:
Should this be in here? I wonder if replication is the only thing that this affects? Shouldn't this be in the Node struct?

Reply:
`Node` struct needs to be `clonable` and for `async` ops.
hence get it held here.

If other places need to utilize it as well in future, some infrastructure refactor might be required.